src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -241,146 +291,220 @@ private void AddOptionIfAvailable<T>(Command command, OptionInfo<T> option)
                 }
 
                 return default;
-            });
-    }
+            };
 
-    /// <summary>
-    /// Static members for <see cref="Driver{TCallable,TIn,TOut}"/>.
-    /// </summary>
-    internal static class Driver
-    {
-        // TODO: Define the aliases as constants.
+            command.AddValidator(validator);
+            return new[] { validator };
+        }
 
         /// <summary>
-        /// The subscription option.
+        /// Creates the simulate command.
         /// </summary>
-        internal static readonly OptionInfo<string> SubscriptionOption = new OptionInfo<string>(
-            ImmutableList.Create("--subscription"), "The subscription ID.");
+        /// <param name="entryPointCommands">The entry point commands that will be the sub commands to the created command.</param>
+        /// <returns>Tuple with the created simulate command and the validators for that command.</returns>
+        private static (Command, IEnumerable<ValidateSymbol<CommandResult>>) CreateSimulateCommand(
+            IEnumerable<(Command, IEnumerable<ValidateSymbol<CommandResult>>)> entryPointCommands)
```

Comment:
Since this pair of command and validators is used in several places (and the list of validators on a command is unfortunately internal to System.CommandLine), I recommend creating a new type to hold them instead of using a tuple.

Reply:
done